department-of-veterans-affairs/caseflow
# Replace ActiveJob queue adapter with shoryuken (instead of sidekiq)

Part 3 of #3182. 

At this point, the shoryuken gem is installed and shoryuken workers are running on the environments.  This turns SQS as the queue backend for ActiveJobs. 

**To Test**
```
$ rails c
> HeartbeatTasksJob.perform_later
```
 - an outstanding message should end up in an sqs queue (visible [here](https://console.amazonaws-us-gov.com/sqs/home?region=us-gov-west-1#view-messages:selected=https://sqs.us-gov-west-1.amazonaws.com/008577686731/development_low_priority;noRefresh=true;prefix=$))

- Run the shoryuken worker
```
$ shoryuken start -q development_high_priority development_low_priority -R
```
